compiler/main/driver.cpp

```diff
@@ -1806,6 +1808,9 @@ int main(int argc, char* argv[]) {
     process_args(&sArgState, argc, argv);
 
     setupChplGlobals(argv[0]);
+    if (fDynoCompilerLibrary)
+      // set the config params we parsed earlier
```

Comment:
here by "parsed" I think you mean from "process_args" or something ? Since we didn't run the Chapel parser on it (just "parsed" the command line arguments?). So I think the comment should not use the term "parse" here.